quisby/util.py

```diff
@@ -85,28 +85,41 @@ def mk_int(string):
     else:
         return 1
 
+def percentage_deviation(item1,item2):
```

Comment:
When ever we have a value of 0, we need to flag that as being an issue.   A value of 0 most likely means part of a test did not run properly.

Reply:
Err wait, maybe not, I was thinking you were commenting on the % not the individual results.  I agree I can't think of any results that should be 0.0 but at the same time I can't rule it out and I'm not keen on considering something a magic number.